File-Formats/PDF/Working-with-Annotations.md

```diff
@@ -5098,6 +5098,219 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("polygonAnnotation.pdf"
 
 {% endtabs %}
 
+### Watermark Annotation
+
+A watermark annotation is used to represent graphics that are expected to be printed at a fixed size and position on a page, regardless of the dimensions of the printed page., [PdfWatermarkAnnotation](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Interactive.PdfWatermarkAnnotation.html) can be used.
+The following code example explains how to add a watermark annotation in the PDF document
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Load the existing PDF document
+
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument("input.pdf");
+
+ //Get the page 
+ 
+PdfLoadedPage lpage = loadedDocument.Pages[0] as PdfLoadedPage;
+
+//Creates PDF watermark annotation 
+
+PdfWatermarkAnnotation watermark = new PdfWatermarkAnnotation(new RectangleF(50, 100, 100, 50));
+
+ //Sets properties to the annotation 
+ 
+watermark.Opacity = 0.5f; 
+
+//Create the appearance of watermark 
+
+watermark.Appearance.Normal.Graphics.DrawString("Watermark Text", new PdfStandardFont(PdfFontFamily.Helvetica, 20), PdfBrushes.Red, new RectangleF(0, 0, 200, 50), new PdfStringFormat(PdfTextAlignment.Center, PdfVerticalAlignment.Middle));
+
+//Adds the annotation to page 
+
+lpage.Annotations.Add(watermark); 
+
+//Saves the document to disk. 
+
+loadedDocument.Save("WatermarkAnnotation.pdf"); 
+
+loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+
+'Load the existing PDF document
+
+ Dim loadedDocument As New PdfLoadedDocument("input.pdf");
+ 
+'Get the page 
+Dim lpage As PdfLoadedPage = TryCast(loadedDocument.Pages(0),PdfLoadedPage)
+
+'Creates PDF watermark annotation
+
+Dim watermark As New PdfWatermarkAnnotation(New RectangleF(50, 100, 100, 50)) 
+
+watermark.Opacity = 0.5f; 
+
+'Creates the appearance of watermark
+
+watermark.Appearance.Normal.Graphics.DrawString("Watermark Text", New PdfStandardFont(PdfFontFamily.Helvetica, 20), PdfBrushes.Red, New RectangleF(0, 0, 200, 50), New PdfStringFormat(PdfTextAlignment.Center, PdfVerticalAlignment.Middle))
+
+'Adds annotation to the page 
+
+lpage.Annotations.Add(watermark)
+
+'Saves the document to disk. 
+ 
+loadedDocument.Save("WatermarkAnnotation.pdf")
+ 
+loadedDocument.Close(True)
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+
+//Load the existing PDF document
+
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument("input.pdf");
```

Comment:
We can't directly load the PDF document using file in UWP platform, Check our UG and create based on that.